blessed/sequences.py

```diff
@@ -339,36 +355,38 @@ def init_sequence_patterns(term):
             # recognize blue_on_red, even if it didn't cause it to be
             # generated.  These are final "ok, i will match this, anyway" for
             # basic SGR sequences.
-            re.escape(u'\x1b') + r'\[(\d+)\;(\d+)\;(\d+)\;(\d+)m',
-            re.escape(u'\x1b') + r'\[(\d+)\;(\d+)\;(\d+)m',
-            re.escape(u'\x1b') + r'\[(\d+)\;(\d+)m',
-            re.escape(u'\x1b') + r'\[(\d+)?m',
-            re.escape(u'\x1b(B'),
+            ('?', re.escape(u'\x1b') + r'\[(\d+)\;(\d+)\;(\d+)\;(\d+)m'),
```

Comment:
presumably ``'?'`` is, "what should we name it? I would just call it "SGR1" or "SGR4", where the number indicates number of parameters.